apps/website/src/pages/admin/show-and-tell/review/[entryId]/preview.tsx

```diff
@@ -30,7 +30,10 @@ export async function getServerSideProps(context: NextPageContext) {
 
 const AdminPreviewShowAndTellPage: NextPage<
   InferGetStaticPropsType<typeof getServerSideProps>
-> = ({}) => {
+> = (
+  // biome-ignore lint/correctness/noEmptyPattern:
+  {},
```

Comment:
Why are we ignoring the linter here?

Reply:
The rule disallows the use of the empty pattern. I didn't know how it could be done differently.

Reply:
Why can't we just remove the `{}` here, it seems to be doing nothing?